json-path/src/main/java/com/jayway/jsonpath/internal/path/CompiledPath.java

```diff
@@ -117,6 +117,7 @@ public EvaluationContextImpl evaluate(Object rootObj, Object document, Object ro
             PathRef op = ctx.forUpdate() ? PathRef.createRoot(rootDocument) : PathRef.NO_OP;
             root.evaluate("", op, document, ctx);
         } catch (EvaluationAbortException abort) {
+        	throw new RuntimeException(abort);
```

Comment:
Why do you need this? not a good idea to throw runtime here, will kill our jett in the listener jvm